renpy/translation/__init__.py

```diff
@@ -945,7 +945,7 @@ def detect_user_locale():
         if isinstance(locale_name, bytes):
             locale_name = locale_name.decode("utf-8")
 
-        local_name = locale_name.replace("-", "_")
```

Comment:
This should be checked. The `local_name` variable wasn't used, which is weird, but this assignment on the `locale_name` variable changes the behavior of the function.